src/main/modules/nunjucks/index.ts

```diff
@@ -20,11 +20,22 @@ export class Nunjucks {
     });
 
     env.addGlobal('formItems', function (items: FormInput[]) {
-      return items.map(i => ({
-        text: this.env.globals.getContent.call(this, i.label),
-        value: i.value,
-        selected: i.selected,
-      }));
+      return items.map(i => {
+        let conditional: string | undefined;
+        if (i.warning) {
+          conditional = env.render(`${__dirname}/../../steps/common/error/warning.njk`, {
+            message: this.env.globals.getContent.call(this, i.warning),
+            warning: this.ctx.warning,
+          });
+        }
+
+        return {
+          text: this.env.globals.getContent.call(this, i.label),
+          value: i.value,
+          selected: i.selected,
+          conditional: conditional && { html: conditional },
```

Comment:
personally I think I would have done:

```
conditional: !i.warning ? undefined : env.render(`${__dirname}/../../steps/common/error/warning.njk`, {
            message: this.env.globals.getContent.call(this, i.warning),
            warning: this.ctx.warning,
          });
```

Reply:
👍 That does look better, fixed in d92f164